source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,571 @@
+
+Next your users would need to install the mobile application on their device. If desired, you can further configure security capabilities using an EMM provider.
+
+**********************************
+Updating via the Public Store Apps
+**********************************
+
+While not part of your initial mobile deployment, you should consider a strategy for updating when new versions of the Mattermost Mobile applications are available. Simultaneously, you should check any compatibility requirements for the mobile apps and the Mattermost server. 
+
+It is often easier to upgrade the mobile apps. However, not all provided updates are compatible with all previous versions of the Mattermost server. 
+
+    **Note:** Only updating the mobile apps, or updating the mobile apps before the Mattermost server, can result in incompatibility.
+
+----
+
+########################################
+Mobile Deployment via Custom Mobile Apps
+########################################
+
+Choosing this model means you’ve decided not to use the mobile applications Mattermost has made available via the public app stores. 
+
+This also means you will need to maintain these applications. Maintenance includes rebuilding and incorporating feature and/or security updates. Otherwise, your applications will not match the functionality of our publicly available applications, and could be incompatible with future versions of the Mattermost server.
+
+Finally, this choice also means you cannot use the hosted version of the Mattermost Push Notification Service (HPNS).
+
+A great first step for this deployment model involves `reviewing the documentation <https://developers.mattermost.com/contribute/mobile/>`_. Your development team should run through the requirements for building the applications. This will help understand the overall impact when choosing this option. This documentation provides guidance on building, compiling, and signing. It also includes information for customizing the apps.
+
+We’ve provided some recommended sections below.
+
+- Developer Setup
+- Build Your Own App
+- Push Notifications
+
+*********************************
+Distribution through an app store
+*********************************
+
+Once you have built your own apps you will need to distribute them. There are two options:
+
+Option 1 - Enterprise App Store via EMM Provider (Most Common): 
+
+- This is the most common way for customers to distribute their apps.
+- Once added to your own "Enterprise App Store", user can download.
+
+Option 2 - Apple App Store & Google Play (not recommended):
+
+- To submit an app to the official app stores, you need to submit the app to Apple/Google for review.
+- This is the same process Mattermost uses make the apps available for everyone.
+- This process is more common if you are looking to white label the app to remove Mattermost branding.
+- This is not the typical approach.
```

Comment:
why not?

Reply:
This was requested by SAs. I’ll yield to @johnthompson365 on this one. But we made a concerted effort that app customization was not what most customers (within the audience of this guide) would do. I’m ok keeping it unless there is strong opinion or I misunderstood.